internal/services/consumption/consumption_budget_base.go

```diff
@@ -153,13 +152,10 @@ func (br consumptionBudgetBaseResource) arguments(fields map[string]*pluginsdk.S
 						ValidateFunc: validation.IntBetween(0, 1000),
 					},
 					// Issue: https://github.com/Azure/azure-rest-api-specs/issues/16240
```

Comment:
Looks like this comment should also be removed?

Reply:
updated